release-notes/README.md

```diff
@@ -4,755 +4,19 @@ description: Release notes and change logs for the Adobe Experience Platform Mob
-
-## January 19, 2021
-
-### Adobe Experience Platform iOS Core SDKs
-
-Adobe Experience Platform Mobile SDKs for iOS are now being made available on Swift. Mobile Core in Swift is available as an [open sourced project on Github](https://github.com/adobe/aepsdk-core-ios/) and contains the following extensions:
-
-* AEPCore 3.0.0
-* AEPServices 3.0.0
-* AEPIdentity 3.0.0
-* AEPSignal 3.0.0
-* AEPLifecycle 3.0.0
-* AEPRulesEngine 1.0.0
-
-### iOS AEPEdge 1.0.0
-
-Initial release of Adobe Experience Platform Edge Network mobile extension iOS in Swift.
-
-This extension allows for sending XDM formatted data to Adobe Experience Platform and Adobe Experience Cloud solutions, by leveraging Edge Network capabilities.
-
-The included features with this release are:
-
-* Ability to create XDM Experience Events and send them to Experience Edge Network. An optional Experience Edge response callback can be registered per event.
-* The XDM Experience Events can be sent to individual Adobe Experience Platform datasets when a custom dataset identifier is set at event level.
-* Events persistence.
-* ECID is automatically attached on each XDM Experience Event request.
-* Integration with AEPAssurance extension enabling new insights in Project Griffon about the XDM Experience Event processing and XDM data validation for an improved validation and debugging experience.
-* Detailed warning/error messages are available through Project Griffon UI and logs.
-
-This extension library is available as an open sourced project \([adobe/aepsdk-edge-ios](https://github.com/adobe/aepsdk-edge-ios)\) on GitHub.
-
-These iOS libraries, written in Swift, are available for integration via SPM, XCFramework and Cocoapods and can be used in Swift and Objective-C applications.
-
-### Android Edge 1.0.0
-
-The Adobe Experience Platform Edge Network \(Edge\) mobile extension is now available in Android! This extension allows for sending XDM formatted data to Adobe Experience Platform and Adobe Experience Cloud solutions, by leveraging Experience Edge capabilities.
-
-The included features with this release are:
-
-* Ability to create XDM Experience Events and send them to Experience Edge Network. An optional Experience Edge response callback can be registered per event.
-* The XDM Experience Events can be sent to individual Adobe Experience Platform datasets when a custom dataset identifier is set at event level.
-* ECID is automatically attached on each XDM Experience Event request.
-* Integration with Assurance extension enabling new insights in Project Griffon about the XDM Experience Event processing and XDM data validation for an improved validation and debugging experience.
-* Detailed warning/error messages are available through Project Griffon UI and logs.
-
-This SDK is compatible with Android Core 1.5.7 and above.
-
-### Adobe Experience Platform Edge Network Launch Extension 1.0.2
+Upgrade the following flutter package to reference Android libraries from mavenCentral :
```

Comment:
Updated the following flutter packages to reference the Android libraries from Maven Central Repository: